docs/api.md

```diff
@@ -408,6 +410,69 @@ Sets a default zoom factor to be used when the app is opened, defaults to `1.0`.
 
 Prevents application from being run multiple times. If such an attempt occurs the already running instance is brought to front.
 
+#### [assets]
+
+```
+--assets
+```
+
+Allow to specify additional assets array dedicated to the desktop version of your electron application, might be either a file or a directory
+
+Example:
+
+```bash
+nativefier http://google.com --assets ./assets/mainInjector.js --assets ./images ....
```

Comment:
shouldn't the command ordering have options first? i.e.  `nativefier --assets ... http://google.com`